admin/src/helpers/internal-fetch.js

```diff
@@ -0,0 +1,26 @@
+import { merge } from 'lodash/object';
+import session from './Session';
+
+const internalFetch = (url, config = {}) => {
```

Comment:
@Jakedavies after you rebase can you refactor to use internalFetch on the changes I submitted?

Reply:
will do